tests/autofix_files/trio100.py.diff

```diff
@@ -1,6 +1,6 @@
 ---
 +++
-@@ -2,24 +2,24 @@
+@@ -3,24 +3,24 @@
```

Comment:
We might want to replace these numbers with `x` so that inserting or deleting unchanged code doesn't affect the diff?  Definitely a follow-up PR if we decide to do that though.

Reply:
ye it's very ugly - but wasn't sure if they might still be useful. But I guess if you actually want to find the line numbers you can look at the non-diff autofix file or so.